src/vs/platform/quickinput/browser/quickInput.ts

```diff
@@ -1416,6 +1418,12 @@ export class QuickInputController extends Disposable {
 						}
 					}
 					break;
+				case KeyCode.Space:
+					if (event.ctrlKey) {
```

Comment:
@TylerLeonhardt there's a couple of potential issues with this solution regarding keyboard accessibility.

In order to be discoverable, this should either be conveyed to the user through markup (as in, "has tooltip, press [ .. ] to activate". or at least be a well-known key combination across users (such as [Shift+F1](https://github.com/w3c/aria-practices/issues/811#issuecomment-525414253)).

Additionally note that Ctrl is actually the key that's commonly used to _dismiss_ tooltips rather than summon them in many Windows applications and [is preferred by many keyboard users](https://youtu.be/lb0_v7D4kbs?t=1548) due to that.

Reply:
probably should be resolved somehow in https://github.com/microsoft/vscode/issues/175662